package.json

```diff
@@ -183,8 +183,9 @@
     "package-json/got": "^11.8.5",
     "http-cache-semantics": "~4.1.1",
     "cacheable-request": "^10.2.7",
-    "@mui/base@5.0.0-alpha.98": "patch:@mui/base@npm%3A5.0.0-alpha.98#./.yarn/patches/@mui-base-npm-5.0.0-alpha.98-f4d605d753.patch",
-    "@mui/x-data-grid@^5.17.4": "patch:@mui/x-data-grid@npm%3A5.17.4#./.yarn/patches/@mui-x-data-grid-npm-5.17.4-542730a19e.patch"
+    "@mui/x-data-grid@^5.17.4": "patch:@mui/x-data-grid@npm%3A5.17.4#./.yarn/patches/@mui-x-data-grid-npm-5.17.4-542730a19e.patch",
+    "@mui/base@^5.0.0-beta.36": "patch:@mui/base@npm%3A5.0.0-beta.40#./.yarn/patches/@mui-base-npm-5.0.0-beta.40-248417914d.patch",
+    "@mui/base@5.0.0-beta.40": "patch:@mui/base@npm%3A5.0.0-beta.40#./.yarn/patches/@mui-base-npm-5.0.0-beta.40-248417914d.patch"
```

Comment:
Could you explain why you need 2 `@mui/base` resolutions? I see that one is `.36` and the other is `.40`. I assume our different MUI packages use different versions of `@mui/base`.

Reply:
`yarn` creates these resolutions automatically when you commit the patch. I believe you're right. `@mui/x-date-pickers` depends on `^5.0.0-beta.36` and `@mui/material` depends on `5.0.0-beta.40`, so I believe that this makes sure that all versions of `@mui/base` are patched. It also created a resolution for `@mui/lab`, which depends on `5.0.0-alpha.98`, that I must have removed, so I'm going to add that one back too.

Reply:
> It also created a resolution for `@mui/lab`, which depends on `5.0.0-alpha.98`, that I must have removed, so I'm going to add that one back too.

I forgot that that resolution causes build/test issues, and that's why I removed it.